### PR TITLE
DEV2-7020: single tenant SAAS 

### PR DESCRIPTION
**Pull request by amirbilu**

### CODE REVIEW

File: modules/cluster/cloud_router.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module "cloud_router" {
   source  = "terraform-google-modules/cloud-router/google"
-  version = "~> 5.0.1"
+  version = "6.0.2"
```
Comment (garyb9) [inline modules/cluster/cloud_router.tf:3]: this falls on validate

File: modules/cluster/gke.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module "gke" {
-  source                      = "terraform-google-modules/kubernetes-engine/google//modules/beta-private-cluster"
-  project_id                  = var.project_id
-  kubernetes_version          = "1.27.3-gke.100"
+  source     = "terraform-google-modules/kubernetes-engine/google//modules/beta-private-cluster"
+  project_id = var.project_id
+  # kubernetes_version          = "1.27.3-gke.100"
```
Comment (amirbilu) [inline modules/cluster/gke.tf:4]: set statically to latest

Comment (garyb9) [reply modules/cluster/gke.tf]: bumped

File: examples/cluster/main.tf [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // Tabnine cluster module
 module "gke_cluster_tabnine" {
-  source                         = "codota/gke-tabnine/google//modules/cluster"
+  # source                         = "codota/gke-tabnine/google//modules/cluster"
```
[the remark this answers is not in the document]
Comment (garyb9) [reply examples/cluster/main.tf]: reverted

File: modules/cluster/kms.tf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 module "kms" {
   source             = "terraform-google-modules/kms/google"
-  version            = "~> 2.2"
+  version            = "2.3.0"
```
Comment (amirbilu) [inline modules/cluster/kms.tf:3]: ulay ~> 2.3?

**Review by garyb9 — APPROVED** [review]

**Pull request merged**